kubernetes-maven-plugin/doc/src/main/asciidoc/inc/enricher/_jkube_docker-registry-secret.adoc

```diff
@@ -0,0 +1,31 @@
+[[jkube-docker-registry-secret]]
```

Comment:
The content of this is duplicate to the one in section `5.1.2 Secrets`

The Enricher entry is maybe valid in order to show how to activate or de-activate this feature when using profiles. But duplicating content will mislead users + be harder to maintain, just point/link to the right section of the documentation.

Something like:
`This enricher enables the {goal-prefix}:resource Secret feature.`, `You can read more about this feature here.`...